cw-2.py

```diff
@@ -102,10 +132,16 @@ def some_tests():
     any code block...
 Note: Timeout value can be a float.
 '''
+
+
 def timeout(sec):
     def wrapper(func):
         from multiprocessing import Process
-        process = Process(target=func)
+        msg = 'Should not throw any exception inside timeout'
+
+        def wrapped():
+            expect_no_error(msg, func)
+        process = Process(target=wrapped)
```

Comment:
@Bubbler-4 so the changes are adding `expect_no_error` and wrapping `func` with `expect_no_error`. Other lines are just formatting and being explicit (`Exception`), right?

Reply:
That's right. The linter was super angry so I just had to clean it up.

Reply:
Cool, thanks for doing this. I'll let you know when I deploy the fix.